lib/spree/core.rb

```diff
@@ -35,7 +35,7 @@ def self.user_class
   #
   # This method is defined within the core gem on purpose.
   # Some people may only wish to use the Core part of Spree.
-  def self.config(&block)
+  def self.config
```

Comment:
this will blow up if called with parameters, right?

Reply:
no, it won't because of the way blocks in methods work. It will if no block is passed. See below :point_down: 

```ruby
irb(main):001:0> def foo
irb(main):002:1> yield 'hola'
irb(main):003:1> end
=> :foo
irb(main):004:0> foo
LocalJumpError: no block given (yield)
        from (irb):2:in `foo'
        from (irb):4
        from /home/pau/.rbenv/versions/2.3.7/bin/irb:11:in `<main>'
irb(main):005:0> foo { |msg| puts msg }
hola
=> nil
```

Reply:
Awesome, I'll make this change :+1: 
Thanks!!!

Reply:
awesome, thanks!!!